sklearn/preprocessing/label.py

```diff
@@ -147,6 +149,9 @@ def inverse_transform(self, y):
         y : numpy array of shape [n_samples]
         """
         check_is_fitted(self, 'classes_')
+        # inverse transform of empty array is empty array
```

Comment:
please add a comment to transform or remove this one.

Reply:
I am sorry. I didn't get what you mean by "add a comment to transform"?

Reply:
please add a similar comment to function transform (before L130)

Reply:
Oh got it. Thanks

Reply:
done!